src/index.html

```diff
@@ -6,9 +6,59 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
```

Comment:
fix attributes for all links
```suggestion
    <link rel="stylesheet" href="./style.css">
```